src/Padawan/Domain/Project/Node/TypeHint.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Padawan\Domain\Project\Node;
+
+class TypeHint extends Variable {
+    public $startLine = 0;
+
+    public static function create(Variable $variable, $startLine)
```

Comment:
but this static function isn't a type converter, but a factory function that brews `TypeHint` from `Variable`, so IMO it should stay in `TypeHint`